packages/amplify-provider-awscloudformation/src/aws-regions.js

```diff
@@ -14,6 +14,7 @@ const regionMappings = {
   'eu-central-1': 'EU (Frankfurt)',
   'ap-northeast-1': 'Asia Pacific (Tokyo)',
   'ap-northeast-2': 'Asia Pacific (Seoul)',
+  'ap-northeast-3': 'Asia Pacific (Osaka)',
```

Comment:
please read and follow a note from the comment that's at the beginning of this file.

Reply:
My apologies, last commit addresses this.